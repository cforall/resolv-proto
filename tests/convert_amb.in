f-s1s1 3 3
f-s0s2 2 4
f-s0s1 4 2
g T | f T T
%%
// f( 2 2 )
g( 2 )
