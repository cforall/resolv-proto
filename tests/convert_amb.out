
[Void / (0,1,1)]{T@g => 3 | Void f T@g T@g => f-s1s1( 3 3 )} g{T |  f T T}( 2 => 3 )
